pkg/kms/legacykms/kms_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"crypto/rand"
 	"encoding/json"
 	"fmt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/signature/verifier"
```

Comment:
move this import under line 20 below

Reply:
that's why the lint check failed... rebased